README.md

```diff
@@ -2,24 +2,23 @@
 
 The uVisor is a self-contained software hypervisor that creates independent secure domains on ARM Cortex-M3 and M4 micro-controllers. Its function is to increase resilience against malware and to protect secrets from leaking even among different modules of the same application. You can find a [high level overview here](http://www.slideshare.net/FoolsDelight/resilient-iot-security-the-end-of-flat-security-models).
 
-## Overview
-
-To start using uVisor you will need to include it as a library in your design. We release the uVisor library periodically into the mbed OS repository, [ARMmbed/mbed-os](https://github.com/ARMmbed/mbed-os).
-
-If you want to learn more about the uVisor security model and get an overview of its features this is the right place. In this document you will read about:
+To start using uVisor you will need to include it as a library in your design. We release the uVisor library periodically into the mbed OS repository, [ARMmbed/mbed-os](https://github.com/ARMmbed/mbed-os). If you want to learn more about the uVisor security model and get an overview of its features this is the right place. In this document you can read about:
 
 * The uVisor design philosophy.
 * A technical overview of the uVisor features:
     * Memory layout.
     * Secure boot.
     * Context switching.
 
-You will find the full uVisor documentation in the [docs](docs) folder – see our [OS-level introduction](https://github.com/ARMmbed/uvisor/raw/docs/uvisor-rtos-docs.pdf) in PDF format.
-
-To see how uVisor is used in practice, we created a [basic uVisor example](https://github.com/ARMmbed/mbed-os-example-uvisor) to show how secure interrupts and C++ objects are instantiated in the context of secure boxes. In our [uVisor threaded example](https://github.com/ARMmbed/mbed-os-example-uvisor-thread) we demonstrate the configuration of multiple boxes containing secure threads.
+You can find most of the uVisor documentation in the [docs](docs) folder. Please have a look at our [quickstart guide](docs/api/QUICKSTART.md) for an introduction into uVisor application devlopment. If you are  interested in uVisor internals please refer to the [OS-level introduction](https://github.com/ARMmbed/uvisor/raw/docs/uvisor-rtos-docs.pdf) and our [uVisor API docs](docs/api/API.md).
 
 Contributions to this repository in the form of issue reporting or pull requests are welcome! Please make sure to read our [contribution guidelines](CONTRIBUTING.md) first.
 
+### Getting Started Examples
+* The [basic uVisor example](https://github.com/ARMmbed/mbed-os-example-uvisor) shows how secure interrupts and C++ objects are instantiated in the context of secure boxes.
```

Comment:
Can you add a newline between the title and the first bullet point? It ensures this markdwon renders fine with stricter parsers.